Test-czn.gocd.yaml

```diff
@@ -109,34 +117,79 @@ pipelines:
           type: success
           allow_only_on_success: false
         jobs:
-          deployToDev:
+          smoke-test:
             timeout: 0
             resources :
             - docker
             tasks:
+            - fetch:
+                pipeline: Test
+                stage: Test
+                job: unit-test
+                source: gitCommit.txt
+                is_file: true
+                run_if: passed
             - fetch: 
                 pipeline: Test
                 stage: PackageAndPublish
                 job: docker
                 source: docker-compose-czn.yaml
                 is_file: true
                 run_if: passed
+            - fetch: 
+                pipeline: Test
+                stage: PackageAndPublish
+                job: docker
+                source: .env
+                is_file: true
+                run_if: passed
             - script: |
-                echo deploy here
-                ls -la
-                export TAG_VERSION=${GIT_COMMIT_HASH} && docker-compose -f docker-compose-czn.yaml up -d
-    - Smoke:
+                export TAG_VERSION=$(cat gitCommit.txt) && docker-compose -f docker-compose-czn.yaml up -d
+  Capacity:
+    group: AT20-czn
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      TARGET_HOST: 192.168.0.18
+    materials:
+      Test:
+        pipeline: Test
```

Comment:
Update the material to get from previous pipeline. If smoke passes, then continue with the capacity for the next PR